website/docs/r/alert_channel_microsoft_teams.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_microsoft_teams"
+description: |-
+  Create and manage Microsoft Teams Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_microsoft_teams
+
+You can configure Lacework to forward alerts to a Microsoft Teams channel through an incoming webhook.
```

Comment:
Could we add a link to https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook

Reply:
Added!